### ISSUE TITLE
Accessibility: image background and alt description

### ISSUE THREAD

**Issue by ariasuni** [satwikkansal/wtfpython#74]
- When reading the Markdown file with an application with dark background, or when using dark background in Github (with browser modules etc.), somes images are not legible.
- Images doesn’t have an `alt=` for people using screen reader (I can do that if you want).

**Comment by satwikkansal** [comment]
> When reading the Markdown file with an application with dark background, or when using dark background in Github (with browser modules etc.), somes images are not legible.

I see, that's probably because some images have a transparent background and black outlines (which kind of mixes with dark themes). Having a white background should fix this I guess, what do you think?

> Images doesn’t have an alt= for people using screen reader (I can do that if you want).

Ah, that's bad. I'd appreciate if you'd like to fix this :)

**Comment by Contrastellar** [comment]
I figured I'd throw my hat in here -- I wouldn't mind tackling this issue (following the ./CONTRIBUTING.md guide). ^.^

**Comment by satwikkansal** [comment]
Hey @Contrastellar, sure, would appreciate your help! :)

**Comment by ariasuni** [comment]
Just noticing this again, more than 4 years later, oops.

> Having a white background should fix this I guess, what do you think?

Some pictures having a kind of greyish background, that could be generalized.

> Ah, that's bad. I'd appreciate if you'd like to fix this :)

Unfortunately I don’t have much time, but hopefully @satwikkansal can do that :slightly_smiling_face:

**Comment by Contrastellar** [comment]
> Just noticing this again, more than 4 years later, oops.

Time doesn't exist during the pandemic, its okay!

> Having a white background should fix this I guess,

I picked something lighter than the background of my GitHub, and something not super bright eitehr

Before --
![image](https://user-images.githubusercontent.com/8060101/172493924-c9597d8f-b0c2-41b2-a571-7fae2d19c4d5.png)

After --
![image](https://user-images.githubusercontent.com/8060101/172493785-7322fce7-c5ea-4254-a5a3-d95ecc09f0a2.png)

If this looks about to what @ariasuni had in mind, I wouldn't mind someone [taking a look](https://github.com/Contrastellar/wtfpython) at the Alt text on my fork before i submit my PR. ^^

**Comment by satwikkansal** [comment]
I love the alt-texts! However, the images are unreadable on the light theme now :) 
<img width="837" alt="image" src="https://user-images.githubusercontent.com/10217535/172531529-d0745387-fe4d-484b-b923-997d13392396.png">

**Comment by Contrastellar** [comment]
Oh wack. I'm about to go out of town this weekend, but I'll take a peek at it when I get back. ^^

**Comment by nifadyev** [comment]
hey @satwikkansal , I have found out that Github now [supports](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to) image source based on chosen theme
I can work on it, if accessibility is still important. Also, most of the images are linear, and it is recommended to save these images as `svg` to increase sharpness and reduce size.

**Comment by satwikkansal** [comment]
Hey @nifadyev, thanks let's use a theme-based image source. And we can use svgs also, great idea!
